fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -249,6 +247,26 @@ public void getTimeMapFromAgWithChildrenWithDifferentVersions() throws Exception
         }
     }
 
+    @Test
+    public void getMementoFromAgChild() throws Exception {
+        final var childId1 = id + "/child1";
+
+        createVersionedArchivalGroup(id);
+        createMemento(subjectUri);
+        TimeUnit.SECONDS.sleep(1);
+
+        putVersionedBinary(childId1, OCTET_STREAM_TYPE, "v2", false);
+        final var mementoUri = createMemento(subjectUri);
+        final var mementoTime = mementoUri.substring(mementoUri.lastIndexOf("/"));
+
+        final HttpGet httpGet = new HttpGet(subjectUri + "/child1/fcr:versions" + mementoTime);
+        try (final CloseableHttpResponse response = execute(httpGet)) {
+            assertMementoDatetimeHeaderMatches(response, now());
```

Comment:
The assertion logic just makes sure that the memento timestamp is within a few seconds of the specified time because it's not possible to specify a timestamp anymore.